src/Symfony/Component/Validator/Constraints/FileValidator.php

```diff
@@ -170,12 +172,12 @@ public function validate($value, Constraint $constraint)
         }
 
         if ($constraint->mimeTypes) {
-            if (!$value instanceof FileObject) {
-                $value = new FileObject($value);
+            if (!class_exists(MimeTypes::class)) {
+                throw new LogicException('Validating mime types requires the "Mime" component. Install "symfony/mime" to use it.');
```

Comment:
let's reuse the template given by existing messages, e.g.:
`You cannot validate the mime-type of files as the Mime component is not installed. Try running "composer require symfony/mime".`